src/components/TodoComponents/Todo.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import TodoForm from './TodoForm'
+import TodoList from './TodoList'
+class Todo extends Component{
```

Comment:
The idea behind the `Todo.js` file is to serve as a component for individual todo items in the list (which is why it was recommended by the instructions comments to place the event handlers in `App.js`).